stellar-contract-env-host/src/vm.rs

```diff
@@ -142,4 +148,21 @@ impl VM {
             ))
         }
     }
+
+    /// Invoke a function in the VM's module, converting externally stable
+    /// XDR ScVal arguments into Host-specific RawVals and converting the
+    /// RawVal result back to an ScVal.
+    pub fn invoke_function(
+        &self,
+        host: &mut Host,
+        func: &str,
+        args: &ScVec,
+    ) -> Result<ScVal, Box<dyn Error>> {
+        let mut raw_args: Vec<RawVal> = Vec::new();
+        for scv in args.0.iter() {
```

Comment:
This looks annoying that you need to always be `.0`ing on types that wrap Vec's in the XDR. I'll surface impls of Vec's API up into all types that wrap a Vec to make it better.

Reply:
Created issue for this: https://github.com/stellar/rs-stellar-xdr/issues/45

Reply:
Doesn't matter to me at all, but I won't stop you if it bugs you!